tests/unit/Codeception/Module/AMQPTest.php

```diff
@@ -1,55 +1,101 @@
 <?php
 
+namespace App\Tests\unit\Codeception\Module;
 
-class AMQPTest extends \Codeception\PHPUnit\TestCase
+use Codeception\Exception\ModuleConfigException;
+use Codeception\Exception\ModuleException;
+use Codeception\Lib\ModuleContainer;
+use Codeception\Module\AMQP;
+use Codeception\PHPUnit\TestCase;
+use Codeception\Util\Stub;
+use Exception;
+use PhpAmqpLib\Exception\AMQPProtocolChannelException;
+use PhpAmqpLib\Message\AMQPMessage;
+
+/**
+ * Class AMQPTest
+ * @package App\Tests\unit\Codeception\Module
+ */
+final class AMQPTest extends TestCase
 {
-    protected $config = array(
+    protected const QUEUES    = [
+        'queue1',
+        'queue2',
+        'queue3',
+        'test-queue',
+    ];
+    protected const EXCHANGES = [
+        'test-exchange',
+    ];
+
+    protected $config = [
         'host'     => 'localhost',
         'username' => 'guest',
         'password' => 'guest',
-        'port' => '5672',
+        'port'     => '5672',
         'vhost'    => '/',
-        'cleanup' => false,
-        'queues' => array('queue1')
-    );
+        'cleanup'  => false,
+        'queues'   => ['queue1'],
+    ];
 
     /**
-     * @var \Codeception\Module\AMQP
+     * @var AMQP|null
      */
-    protected $module = null;
+    protected $module;
 
-    public function _setUp()
```

Comment:
`_setUp` is Codeception wrapper for PHPUnit's `setUp` method, because the signature has changed in PHPUnit 8.0.